app/src/main/res/values/component_colors.xml

```diff
@@ -62,6 +62,7 @@
   <color name="component_color_shared_text_input_layout_error_color">@color/color_palette_error_color</color>
   <color name="component_color_general_item_background_solid_color">@color/color_palette_container_background_color</color>
   <color name="component_color_general_item_background_stroke_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_general_item_secondary_background_stroke_color">@color/Color_palette_stroke_color</color>
```

Comment:
Typo? Does this actually work with the capitalized ``Color``?

Reply:
Thanks @BenHenning , corrected it